pkg/sysregistriesv2/shortnames.go

```diff
@@ -0,0 +1,274 @@
+package sysregistriesv2
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/containers/image/v5/docker/reference"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/storage/pkg/homedir"
+	"github.com/containers/storage/pkg/lockfile"
+	"github.com/pkg/errors"
+)
+
+// defaultShortNameMode is the default mode of registries.conf files if the
+// corresponding field is left empty.
+const defaultShortNameMode = types.ShortNameModePermissive
+
+// userShortNamesFile is the user-specific registries.conf file to store aliases.
+var userShortNamesFile = filepath.FromSlash(".cache/containers/short-name-aliases.conf")
```

Comment:
Could you use `$XDG_CACHE_HOME` if set? It is the [defined convention](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), and would make testing easier/safer as well.

Reply:
Sounds good. I'll update the code (and the docs).